pkg/acsengine/const.go

```diff
@@ -104,8 +104,8 @@ const (
 	DefaultACIConnectorAddonName = "aci-connector"
 	// DefaultDashboardAddonName is the name of the kubernetes-dashboard addon deployment
 	DefaultDashboardAddonName = "kubernetes-dashboard"
-	// DefaultACIConnectorImage defines the ACI Connector deployment version on Kubernetes Clusters
-	DefaultACIConnectorImage = "virtual-kubelet:latest"
```

Comment:
Did we want to delete this const?

Reply:
yea, this one is no longer used (since it moved to `k8s_versions`), thought i would clean it up